### PR TITLE
Image-classification and Object detection Online cli and SDK files to…

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
… pick latest model

# Description

# Checklist

- [ ] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] Pull request includes test coverage for the included changes.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by KonakanchiSwathi** [comment]
@rjaincc  @MadhuM02  can you please review thank in  advance

File: cli/foundation-models/system/inference/image-object-detection/image-object-detection-online-endpoint.sh [hunk 1]
```diff
@@ -52,14 +52,19 @@ az ml online-endpoint create --name $endpoint_name $workspace_info  || {
     echo "endpoint create failed"; exit 1;
 }
 
-# deploy model from registry to endpoint in workspace
-az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
-  endpoint_name=$endpoint_name \
-  model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
-  instance_type=$deployment_sku || {
-    echo "deployment create failed"; exit 1;
+# Deploy model from registry to endpoint in workspace
```
Comment (rjaincc) [inline cli/foundation-models/system/inference/image-object-detection/image-object-detection-online-endpoint.sh:55]: Let's make the change for instance segmentation as well at the same time.

Comment (KonakanchiSwathi) [reply cli/foundation-models/system/inference/image-object-detection/image-object-detection-online-endpoint.sh:55]: ok

Comment (KonakanchiSwathi) [reply cli/foundation-models/system/inference/image-object-detection/image-object-detection-online-endpoint.sh:55]: Updated Segmentation files @rjaincc  can you please review Thanks in advance

**Pull request closed without merging**